prime-dotnet-webapi/Configuration/Database/Seeded Tables/CollegeLicenseConfiguration.cs

```diff
@@ -45,7 +45,6 @@ public override IEnumerable<CollegeLicense> SeedData
                     new CollegeLicense { CollegeCode = 1, LicenseCode = 89 },
                     new CollegeLicense { CollegeCode = 1, LicenseCode = 90 },
                     new CollegeLicense { CollegeCode = 1, LicenseCode = 91 },
-                    new CollegeLicense { CollegeCode = 1, LicenseCode = 92 },
```

Comment:
Eliminating this (for Certified Physician Assistant) doesn't seem correct?

Reply:
Good catch! It is fixed and the database has been rebuilt.